ExploreASL_Process.m

```diff
@@ -85,6 +85,14 @@
 
 		% Iterating across all subjects
         [~, x] = xASL_init_Iteration(x,'xASL_module_BIDS2Legacy');
+
+		% Run cleanup
```

Comment:
How is this a "cleanup"?

Reply:
Yes, a name Michael used. I'll rename this to finalize.